# Terrain state definition
Header header

AtomicProposition[]  terrain_state